lib/fluent/plugin/in_tail.rb

```diff
@@ -92,6 +88,8 @@ def configure(conf)
 
       configure_parser(conf)
       configure_tag
+      @encoding = parse_encoding_param(conf['encoding'])
+      @from_encoding = parse_encoding_param(conf['from_encoding'])
```

Comment:
It's bad practice to get values by `conf[]` reference, because it's not affected from type conversion and default value configurations. It might be done by subclasses (3rd party plugin which inherits `in_tail`).